### PR TITLE
Feat : Do not use AWS CLI command

### PR DESCRIPTION
**Pull request by GiliFaroEnv0**
Solve issue https://github.com/env0/k8s-modules/issues/5

### CODE REVIEW

File: aws/autoscaler/main.tf [hunk 1]
```diff
@@ -43,23 +43,20 @@ module "eks-cluster-autoscaler" {
...
+    enabled_metrics       = ['GroupMinSize',
+      'GroupMaxSize',
+      'GroupDesiredCapacity',
+      'GroupInServiceInstances',
+      'GroupPendingInstances',
+      'GroupStandbyInstances',
+      'GroupTerminatingInstances',
+      'GroupTotalInstances']
+  }
 }
```
Comment (eranelbaz) [inline aws/autoscaler/main.tf]: where `granularity \"1Minute\"`?

**Review by yaronya — CHANGES_REQUESTED** [review]
@GiliFaroEnv0 did you test it? Doesn't look like the autoscaler module has a `workers_group_defaults` attribute

I think it should be under `aws/eks/main.tf`

**Comment by GiliFaroEnv0** [comment]
[Do not use AWS CLI command #5](https://trello.com/c/p4mbBNQt/5341-do-not-use-aws-cli-command-5)

**Comment by roni-frantchi** [comment]
We'll have to look at a different approach - this one does not work

**Pull request closed without merging**